geolocation/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+android {
+    compileSdkVersion 29
```

Comment:
you must use the dependencies.gradle

Reply:
Podemos meterlo con variable lo de la version 29